### PR TITLE
[Redesign] Update Simple Logger Operators

### PR DESCRIPTION
**Pull request by objorkman**
* Updates to `BoundingBoxLoggerOperator`, `CameraLoggerOperator`, and `MultipleObjectTrackerOperator`

### CODE REVIEW

File: pylot/loggers/bounding_box_logger_operator.py [hunk 1]
```diff
@@ -32,32 +30,25 @@ def __init__(self, obstacles_stream: erdos.ReadStream,
...
+                data: ObstaclesMessageTuple):
         """Logs bounding boxes to files.
 
         Invoked upon the receipt of a msg on the obstacles stream.
 
         Args:
-            msg (:py:class:`~pylot.perception.messages.ObstaclesMessage`):
+            data (:py:class:`~pylot.perception.messages.
+                   ObstaclesMessageTuple`):
                 Received message.
```
Comment (pschafhalter) [inline pylot/loggers/bounding_box_logger_operator.py:42]:
```suggestion
            data: The obstacle to log.
```

File: pylot/loggers/bounding_box_logger_operator.py [hunk 2]
```diff
@@ -20,10 +21,7 @@ class BoundingBoxLoggerOperator(erdos.Operator):
         _msg_cnt (:obj:`int`): Number of messages received.
         _data_path (:obj:`str`): Directory to which to log files.
     """
-    def __init__(self, obstacles_stream: erdos.ReadStream,
-                 finished_indicator_stream: erdos.WriteStream, flags,
-                 file_base_name):
-        obstacles_stream.add_callback(self.on_obstacles_msg)
+    def __init__(self, flags, file_base_name):
```
Comment (pschafhalter) [inline pylot/loggers/bounding_box_logger_operator.py:24]:
```suggestion
    def __init__(self, flags, file_base_name: str):
```

File: pylot/loggers/camera_logger_operator.py [hunk 3]
```diff
@@ -34,21 +35,12 @@ def __init__(self, camera_stream: erdos.ReadStream,
...
-        """Connects the operator to other streams.
-
-        The operator receives a camera stream.
-        """
-        finished_indicator_stream = erdos.WriteStream()
-        return [finished_indicator_stream]
-
-    def on_frame(self, msg: erdos.Message):
+    def on_data(self, context: OneInOneOutContext, data: CameraFrame):
         """Invoked whenever a frame message is received on the stream."""
```
Comment (pschafhalter) [inline pylot/loggers/camera_logger_operator.py:39]:
```suggestion
        """Invoked whenever a camera frame is received."""
```

File: pylot/loggers/multiple_object_tracker_logger_operator.py [hunk 4]
```diff
@@ -32,29 +34,26 @@ def __init__(self, obstacles_stream: erdos.ReadStream,
...
+                data: ObstaclesMessageTuple):
         """Logs obstacles to files.
 
         Invoked upon the receipt of a msg on the obstacles stream.
 
         Args:
-            msg (:py:class:`~pylot.perception.messages.ObstaclesMessage`):
+            data (:py:class:`~pylot.perception.messages.
+                   ObstaclesMessageTuple`):
                 Received message.
```
Comment (pschafhalter) [inline pylot/loggers/multiple_object_tracker_logger_operator.py:46]:
```suggestion
            data: Received obstacles.
```

**Review by pschafhalter — CHANGES_REQUESTED** [review]

**Comment by objorkman** [comment]
Thanks for the comments - updated!

**Pull request merged**